portal/src/components/FilterCategories/FilterCategories.component.js

```diff
@@ -36,76 +37,47 @@ export default {
         this.$forceUpdate()
       }
     },
-    onToggleShowAll(category) {
-      category.showAll = !category.showAll
-      this.$forceUpdate()
-    },
-    /**
-     * Set filter for v-model
-     * @returns {*} - filters
-     */
-    setFilter() {
-      const { categories, publisherdate } = this
-      if (categories) {
-        let filters = []
-
-        const publisherdate_item = this.value.filters.find(
-          item => item.external_id === publisherdate
-        )
-
-        if (publisherdate_item) {
-          filters.push({
-            external_id: publisherdate,
-            items: [...publisherdate_item.items]
-          })
-          this.data = {
-            start_date: publisherdate_item.items[0],
-            end_date: publisherdate_item.items[1]
-          }
-        }
-
-        filters = Object.assign({}, this.value, {
-          filters
-        })
-
-        this.$router.push(this.generateSearchMaterialsQuery(filters))
+    onChange(e) {
+      const { categoryId, itemId } = e.target.dataset
+      const filter = { external_id: categoryId, items: [itemId] }
```

Comment:
Kan je niet meerdere items in een filter hebben?

Reply:
Ja, dat kan met bijvoorbeeld "vakgebieden". Deze filter staat momenteel uit, maar lijkt me een goede. Bovendien is het noodzaak voor CC filters (zie comments)

Reply:
ja maar als je een item aanzet, dan komt er meteen een search call, dus dan is het er maar 1